syllabus/1-Cryptography/6-Advanced_Signatures-slides.md

```diff
@@ -66,13 +71,13 @@ We often want signatures that must be signed<br/>by multiple parties to become v
 
 <pba-flex center>
 
-- Verifier enforced
-- Cryptographic threshold
-- Cryptographic non-threshold<br/>(a.k.a. signature aggregation)
+- Trivial: union of individual signatures.
```

Comment:
Union is often a confusing word for those not mathematically inclined. Can just call it 'sending multiple signatures'.